modules/exploits/linux/local/apt_package_manager_persistence.rb

```diff
@@ -0,0 +1,81 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Local
+  Rank = ExcellentRanking
+  include Msf::Exploit::EXE
+  include Msf::Post::Unix
```

Comment:
Is there a reason for using `Msf::Post::Unix` rather than `Msf::Post::Linux` ?

None of the methods within the `Unix` namespace are used in this module; however the `Unix` mixin also includes `msf/core/post/linux/system`. It should be safe to replace this `include` as follows:

```suggestion
  include Msf::Post::Linux::System
```